pkg/auth/auth.go

```diff
@@ -118,10 +119,11 @@ func Login(ctx context.Context, systemContext *types.SystemContext, opts *LoginO
 	}
 
 	// If no username and no password is specified, try to use existing ones.
+	serverDomain := domain(server)
 	if opts.Username == "" && password == "" && authConfig.Username != "" && authConfig.Password != "" {
-		fmt.Println("Authenticating with existing credentials...")
-		if err := docker.CheckAuth(ctx, systemContext, authConfig.Username, authConfig.Password, server); err == nil {
-			fmt.Fprintln(opts.Stdout, "Existing credentials are valid. Already logged in to", server)
+		fmt.Fprintln(opts.Stdout, "Authenticating with existing credentials for", server)
```

Comment:
Can you use `Fprintf()`?  My brain parser has a hard time eating format arguments without format arguments.

Same below.

Reply:
Fixed as suggested.